src/cromshell/list/command.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import json
+import logging
+
+import click
+import requests
+from tabulate import tabulate
+
+import cromshell.utilities.submissions_file_utils
+from cromshell.utilities import command_setup_utils, cromshellconfig, http_utils
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="list")
+@click.option(
+    "-c",
+    "--color",
+    is_flag=True,
+    default=False,
+    help="Color the output by completion status.",
+)
+@click.option(
+    "-u",
+    "--update",
+    is_flag=True,
+    default=False,
+    help="Check completion status of all unfinished jobs.",
+)
+@click.pass_obj
+def main(config, color, update):
+    """List the status of workflows."""
+
+    LOGGER.info("list")
+
+    # Update the submission database if so requested
+    if update:
+        update_submission_db(config)
+
+    # Iterate over the submissions text database and print to screen in a pretty way
+    with open(cromshellconfig.submission_file_path, "r") as sub_f:
+        reader = csv.reader(sub_f, delimiter="\t", lineterminator="\n")
+        all_table_rows = []
+        for single_table_row in reader:
+            all_table_rows.append(format_status(single_table_row) if color else single_table_row)
```

Comment:
lint test failed, here is the fix
```suggestion
            all_table_rows.append(
                format_status(single_table_row) if color else single_table_row
            )
```